index.js

```diff
@@ -40,34 +40,56 @@ var pick = function(obj, props) {
 };
 
 cli
-  .command('install [input_string]')
+  .command('install <input_strings...>')
   .alias('i')
-  .option('-d, --clean', 'remove node_modules first')
-  .option('--npm', 'use npm to install')
-  .option('-a, --all', 'prune dependencies as well as dev dependencies')
-  .description('install a given subset defined in package.json')
-  .action(function(input_string, options) {
-    if (!input_string) {
-      throw 'Please provide an install subset name';
-    }
+  .option('-d, --clean', 'Remove node_modules first.')
+  .option('--npm', 'Use npm to install.')
+  .option('-a, --all', 'Prune dependencies as well as devDependencies.')
+  .description('Install a given subset or multiple subsets defined in package.json.')
+  .action(function(input_strings, options) {
 
     if (!packageJson.subsets) {
-      throw 'No install subsets in package.json';
+      console.error('No install subsets in package.json.')
+      process.exit(1)
     }
 
-    if (!packageJson.subsets[input_string]) {
-      throw 'No install subset with that name';
+    const subsetStrings = new Array()
+
+    if (input_strings.length >= 1) {
+        for (let string of input_strings) {
+          subsetStrings.push(string)
+        }
     }
 
-    const subset = packageJson.subsets[input_string];
+    let subset = {}
+    for(var i=0; i < subsetStrings.length; i++) {
+      const packageSubset = packageJson.subsets[subsetStrings[i]]
+      if (!packageSubset) {
+        console.error('No install subset with name ' + subsetStrings[i] + ' was found.')
+        process.exit(1)
+      }
+      if (packageSubset.exclude) {
+        if (!subset.exclude) {
+          subset.exclude = []
+        }
+        subset.exclude = [...subset.exclude, ...packageSubset.exclude]
+      }
+      if (packageSubset.include) {
+        if (!subset.include) {
+          subset.include = []
+        }
+        subset.include = [...subset.include, ...packageSubset.include]
+      }
+    }
```

Comment:
Re: my comment in the PR description thread - we should at least have a warning here, possibly an exit condition if both an exclude and include are in the subset object.

Reply:
Agreed